CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Não publicado
+
+### Corrigido
+- Corrigido funcionalidade de `increment` no `fetchListSuccess`, será validado o número da página para evitar duplicidade de dados.
```

Comment:
- Corrigido funcionalidade de `increment` no `fetchListSuccess`, será validado o número da página, caso a página for igual a `1`, então os dados da lista serão resetados, senão será incrementado, isto resolve o problema de quando mudamos de página e voltamos para a mesma e os dados permaneciam os mesmos.

Reply:
feito